### DISCUSSION TITLE
Recipe default and available cultures

### DISCUSSION THREAD

**Discussion by Diman96** [OrchardCMS/OrchardCore#8355] [Q&A] [answered]
Where can i configure  recipe default and availeble cultures?
![image](https://user-images.githubusercontent.com/12182810/105020798-70209880-5a50-11eb-9547-08c342d47363.png)

**Comment by DotCat1985** [comment, marked as answer]
You can configure default and available cultures as follow:

```
{
      "name": "Settings",
      "LocalizationSettings": {
        "DefaultCulture": "ru-RU",
        "SupportedCultures": [ "ru-RU", "uk-UA" ]
      }
}
```

**Reply by hishamco** [reply]
@Skrypt do we support cultures in recipe? I think No, but I'm not sure if there's any updates

**Reply by Skrypt** [reply]
I don't remember sincerely. Probably not. 

Else, I think it's part of SiteSettings Properties. Need to test it ...

Yep that's where it is. So it should work if you add it to your SiteSettings recipe.

**Reply by hishamco** [reply]
> Yep that's where it is. So it should work if you add it to your SiteSettings recipe.

Aha, thanks

**Reply by Skrypt** [reply]
Yeah, test it and let me know because I don't remember if it will import / export these. We just see the "Properties" checkbox but it doesn't tell if the recipe migration code has been implemented.

**Reply by Skrypt** [reply]
I looked quickly and it doesn't seem to work and there is an issue open about it. https://github.com/OrchardCMS/OrchardCore/issues/3966
